certbot/certbot/_internal/snap_config.py

```diff
@@ -93,7 +93,7 @@ def prepare_env(cli_args: List[str]) -> List[str]:
                 CURRENT_PYTHON_VERSION_STRING
             ))
 
-    if len(outdated_plugins) > 0:
+    if len(outdated_plugins):
```

Comment:
```suggestion
    if outdated_plugins:
```

Reply:
https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/

Reply:
oh, I just saw your comment because it got eaten by github when the new code was pushed. I've never seen `if len()` and so it feels weirder to me but again, fully whatever.

Reply:
lol turns out tox agrees: ` C1802: Do not use `len(SEQUENCE)` without comparison to determine if a sequence is empty (use-implicit-booleaness-not-len)`